packages/wallet/detail/identity/walletInfo/index.js

```diff
@@ -18,7 +18,7 @@ const AccountInfo = ({
       <BoxContent className={`${styles.content} ${styles[activeToken]}`}>
```

Comment:
what is this `${styles[activeToken]}` dynamic style ?

Reply:
In the related CSS file, there are different backgrounds depending on the token and the line referred to, helps with that

Reply:
we dont have the btc anymore so make it static

Reply:
@ikem-legend this one it should be change before the merge